charts/kueue/templates/crd/kueue.x-k8s.io_topologies.yaml

```diff
@@ -12,15 +12,7 @@ metadata:
   name: topologies.kueue.x-k8s.io
 spec:
   conversion:
-    strategy: Webhook
-    webhook:
-      clientConfig:
-        service:
-          name: {{ include "kueue.fullname" . }}-webhook-service
-          namespace: '{{ .Release.Namespace }}'
-          path: /convert
-      conversionReviewVersions:
-      - v1
```

Comment:
This is automatically added by https://github.com/kubernetes-sigs/kueue/blob/8b63e005815d8dd48d4d38de0ffe9472d29507c5/hack/update-helm.sh#L75-L89.